src/components/evaluation/EvaluationTable.vue

```diff
@@ -1054,4 +1054,7 @@ tr.border-top-none td {
 .xlisting-note {
   font-size: 0.8em;
 }
+.evaluation-form-btn-cancel {
+  color: rgba(var(--v-theme-on-surface),var(--v-high-emphasis-opacity)) !important;
+}
```

Comment:
@mohamalnadi - This makes sense to me. Can we make it a reusable style in `global.css`? That'll give us an easy-to-use best practice for all such buttons.

Reply:
Sorry, I merged too soon. I agree, but do we actually need a style for it or would just removing `text-primary` be enough?